src/main/java/step1/StringAddCalculator.java

```diff
@@ -0,0 +1,44 @@
+package step1;
+
+import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+    private static final String DEFAULT_DELIMITER = ",|:";
+    private static final String CUSTOM_DELIMITER = "//(.)\n(.*)";
+    private static final int ZERO = 0;
+    private static final int FIRST = 1;
+    private static final int SECOND = 2;
+    private static final SumCalculator SUM_CALCULATOR = SumCalculator.getInstance();
+
+    public static int splitAndSum(String inputString) {
+        if(isEmptyOrNullText(inputString)) {
+            return ZERO;
+        }
+
+        Matcher matcher = Pattern.compile(CUSTOM_DELIMITER).matcher(inputString);
+        if(isCustomString(matcher)) {
```

Comment:
```suggestion
        if (isCustomString(matcher)) {
```
1. 이 부분도 띄어쓰기 한 번 확인해주시면 좋을 것 같아요.
2. 사용자가 정의한 구분자를 포함하는지의 여부를 검사하는 메서드이니, 
이름을 `containsCustomDelimiter` 정도로 지어보면 어떨까요?